av_imu_launch/config/av_gx5_15.yml

```diff
@@ -13,3 +19,550 @@ microstrain_inertial_driver:
 
     # Disable the transform from the mount to frame id transform as it will be handled in the launch file
     publish_mount_to_frame_id_transform : False
+
+    # ******************************************************************
+    # General Settings
+    # ******************************************************************
+
+    # port is the main port that the device will communicate over. For all devices except the GQ7, this is the only available port.
+    # aux_port is only available for the GQ7 and is only needed when streaming RTCM corrections to the device from ROS, or if you want to publish NMEA sentences from this node
+    # port         : "/dev/imu-front"
+    # baudrate     : 230400
+    aux_port     : "/dev/ttyACM1"
```

Comment:
I thought we were supposed to use imu-6 I.e. ttyACM0?

Reply:
Done